src/app/modules/user/user.middleware.ts

```diff
@@ -22,3 +25,15 @@ export const validateContactOtpVerificationParams = celebrate({
     contact: Joi.string().required(),
   }),
 })
+
+/**
+ * Celebrate validation for the update user's last seen feature update date endpoint.
+ */
+export const validateUpdateUserLastSeenFeatureUpdateParams = celebrate({
+  [Segments.BODY]: Joi.object({
+    latestLastSeenFeatureUpdateDate: Joi.date()
+      .format('YYYY-MM-DD')
+      .raw()
```

Comment:
have you checked the format of the date created here? is it utc or some local time? might result in inaccurate comparisons. do you think we should use `Date.now()` timestamp instead?

Reply:
The format of date here is consistent with the format specified in the `format` function and also moment.js format. It will be in UTC format only if the `utc()` function is called.

https://joi.dev/module/joi-date/api/?v=2.1.0

Reply:
I think we can use Date.now() timestamp instead of passing in the date in the request because as long as the viewed date (date at which the update function is called) is later than the latest feature update date, we won't need to notify the form admin users.

Reply:
I prefer a time stamp too; it's less ambiguous. Should update the body prop to timestamp over date then!

Reply:
I changed my implementation to just generate the current date on the server instead of sending it over from the client. This is because it doesn't matter if I update the attribute using the date specified in the JSON file or generate the current date when the update API call was called since both of them will be equal to or later than the date specified in the JSON file.

Reply:
that actually makes much more sense lmao. cheers!